src/file.cpp

```diff
@@ -509,7 +509,14 @@ void SyncFileGet::updatelocalname()
 // add corresponding LocalNode (by path), then self-destruct
 void SyncFileGet::completed(Transfer*, LocalNode*)
 {
-    sync->checkpath(NULL, &localname);
+    LocalNode *ll = sync->checkpath(NULL, &localname);
```

Comment:
I have checked the case in which a file is replaced in the Cloud, and it works correctly